### DISCUSSION TITLE
Device Warning: Ouvislite mini pc

### DISCUSSION THREAD

**Discussion by Dankheadcore** [catchthatrabbit/coreminer#35] [Hardware]
https://www.amazon.ca/gp/product/B09MH72NQR/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&th=1

I bought 3 of these thinking they would hash well. First 2 could not load the factory windows set up or recover. Was going to wipe them and run Ubuntu anyway but it wouldn't load after install. Returned all three. Don't recommend but I think that would be a good SB chipset. 

Also worth noting in my wasted hours I found Ubuntu has a version optimized specificly for Celeron and IOT devices.

**Comment by BA777** [comment]
Hey, thanks for the info. I'm new to mining. 
I'm also looking to purchase a mini computer to use for mining. Raspberry Pi 4's are out of stock everywhere. Any idea what other MC's would work well?
